services/contract_update.proto

```diff
@@ -32,75 +41,93 @@ import "timestamp.proto";
+     * A transaction to update this field is how that rent payment is made.
+     * <p>
+     * This value MUST NOT be less than the current `expirationTime`
+     * of the contract. If this value is earlier than the current
+     * value, the transaction SHALL fail with response
+     * code `EXPIRATION_REDUCTION_NOT_ALLOWED`.
      */
     Timestamp expirationTime = 2;
 
     /**
-     * The new key to control updates to the contract
+     * If set, modify the key that authorizes updates to the contract.
+     * <p>
+     * If this field is set to a valid Key, this key and the previously set key
+     * MUST both sign this transaction.<br/>
+     * If this value is an empty `KeyList`, the prior key MUST sign this
+     * transaction, and the smart contract SHALL be immutable after this
+     * transaction completes, except for expiration and renewal.<br/>
+     * If this value is not an empty `KeyList`, but does not contain any
+     * cryptographic keys, or is otherwise malformed, this transaction SHALL
+     * fail with response code `INVALID_ADMIN_KEY`.
      */
     Key adminKey = 3;
 
     /**
-     * [Deprecated] The new id of the account to which the contract is proxy staked
+     * Replaced with `staked_id` alternatives.
+     * This field is unused and SHALL NOT modify the contract state.<br/>
+     * The id of an account to which the contract is proxy staked
      */
     AccountID proxyAccountID = 6 [deprecated = true];
 
     /**
-     * If an auto-renew account is in use, the lifetime to be added by each auto-renewal.
+     * If set, modify the duration added to expiration time by each
+     * auto-renewal to this value.
      */
     Duration autoRenewPeriod = 7;
 
     /**
-     * This field is unused and will have no impact on the specified smart contract.
+     * This field is unused and SHALL NOT modify the contract state.<br/>
+     * Previously, an ID of a file containing the bytecode of the Solidity
+     * transaction that created this contract.
      */
     FileID fileID = 8 [deprecated = true];
 
-    /**
-     * The new contract memo, assumed to be Unicode encoded with UTF-8 (at most 100 bytes)
-     */
+    // This should be condensed to just a field instead of a oneof and field 9 reserved.
     oneof memoField {
-      /**
-       * [Deprecated] If set with a non-zero length, the new memo to be associated with the account
-       * (UTF-8 encoding max 100 bytes)
-       */
-      string memo = 9 [deprecated = true];
-
-      /**
-       * If set, the new memo to be associated with the account (UTF-8 encoding max 100 bytes)
-       */
-      google.protobuf.StringValue memoWrapper = 10;
+        /**
+         * This value could not accurately distinguish unset or deliberately
+         * empty. memoWrapper should be used instead.<br/>
+         */
+        string memo = 9 [deprecated = true];
+
+        /**
+         * If set, modify the short memo for this smart contract.<br/>
```

Comment:
should the `<br/>` be on the next line ?

Reply:
The syntax guidelines place the `<br/>` tag at the end of the line.  This generally reads better than a tag before text on a line.
e.g.
```html
text ending in a break.<br/>
is generally easier to read than
<br/>text starting with a break
```

Reply:
Added this to guidelines.